src/vs/base/browser/ui/iconLabel/iconLabelHover.ts

```diff
@@ -127,18 +135,28 @@ export function setupCustomHover(hoverDelegate: IHoverDelegate, htmlElement: HTM
 		});
 	};
 	const mouseOverDomEmitter = dom.addDisposableListener(htmlElement, dom.EventType.MOUSE_OVER, () => showHoverDelayed(hoverDelegate.delay), true);
-	return {
+	const hover: ICustomHover = {
 		show: () => {
 			showHoverDelayed(0); // show hover immediately
 		},
 		hide: () => {
 			hideHover(true, true);
 		},
+		update: newTooltip => {
+			markdownTooltip = newTooltip;
+			tooltip = getTooltipForCustom(newTooltip);
+
+			if (hoverWidget && !hoverWidget.isDisposed) {
```

Comment:
I hope this is the right condition to know whether the hover is showing currently.

Reply:
Yes, correct. The hoverWidget either shows or is disposed.